gc-policy.yaml

```diff
@@ -1,15 +1,29 @@
+# Production
+testing:
+    cloud-uploads: 2y
+    containers: 2m
+stable:
+    cloud-uploads: 2y
+    containers: 2m
+next:
+    cloud-uploads: 2y
+    containers: 2m
```

Comment:
@jlebon - thoughts?

Reply:
This SGTM. That said, I wonder if we should send out communications before we start pruning bootimages. We never promised that the bootimages would stick around, but as per [Hyrum's Law](https://www.hyrumslaw.com/), there's guaranteed to be people that rely on them. We shouldn't be afraid to break them, but a heads up would probably be appreciated. WDYT?

Less worried about the container images. We could start pruning those now. 2m WFM for those.

Reply:
Actually, as I was discussing this with @gursewak1997, one thing that I realized is that this probably could break scale up for OKD users due to the classic issue of bootimages never getting updated. Let me reach out to them.